concrete_datastore/api/v1/filters.py

```diff
@@ -29,6 +29,10 @@
 }
 
 JSON_FILTER_PATTERN = r'^\"(?P<str>.*)\"$|(?P<int>^\d+$)|(?P<float>^\d+\.\d+([e][+-]?\d+)?$)|(?P<bool>^true$|^false$)|(?P<null>^null$|^none$)'
+REGEX_DATETIME_MICROSECOND = (
+    "^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(\.\d{1,6})Z$"
```

Comment:
there is no need for a regex group here (`(\.\d{1,6})Z$"`)

you can remove the `(` `)`: `^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{1,6}Z$`